views/github-create-branch-options.hbs

```diff
@@ -0,0 +1,69 @@
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
+  <link rel="stylesheet" href="/public/aui/aui-prototyping.css"
+    integrity="DTM1Q+8lU7SzJT+FWr0JFisCSZlwfM0GiAKYy7h1s9vIKa/CIh37s9NuOCqIOgK4tmqrjLK4NuWuIPUQNsikHA=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="/public/primer/build.css" media="all" />
+  <link rel="stylesheet" href="/public/css/github-create-branch-options.css" media="all" />
+  <link rel="stylesheet" href="/public/css/global.css" media="all" />
+  <link rel="stylesheet" href="/public/css/server-error-message-box.css" media="all" />
+  <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+</head>
+
+<body>
+
+  <section class="gitHubCreateBranchOptions">
+    <div class="headerImage">
+      <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
+    </div>
+    <div class="gitHubCreateBranchOptions__title">
+      <div class="gitHubCreateBranchOptions__header">Create GitHub Branch</div>
+      <div class="gitHubCreateBranchOptions__subHeader">
+        Creating a branch for <b>{{issueKey}}</b>
+      </div>
+    </div>
+
+    <form id="createBranchOptionsForm" class="aui top-label">
+
+      <div class="field-group">
+        <div class="gitHubCreateBranchOptions__optionContainer">
+           {{#if servers.hasCloudServer}}
+            <button class="aui-button gitHubCreateBranchOptions__option gitHubCreateBranchOptions__selected" id="gitHubCreateBranchOptions__cloud">
+              <img src="/public/assets/cloud.svg" class="gitHubCreateBranchOptions__Icon" alt="">
+              GitHub Cloud
+            </button>
+          {{/if}}
+          <button class="aui-button gitHubCreateBranchOptions__option" id="gitHubCreateBranchOptions__enterprise">
+            <img src="/public/assets/server.svg" class="gitHubCreateBranchOptions__Icon" alt="">
+            GitHub Enterprise Server
+          </button>
+        </div>
+      </div>
+      <div class="field-group gitHubCreateBranchOptions__serversContainer">
+        <aui-label for="server">GitHub Enterprise Server instance</aui-label>
+        <select id="ghServers" name="server" class="full-width-field" placeholder="Select a server ">
+          {{#each servers.gheServerInfos}}
+          <option value="{{uuid}}">{{baseUrl}} / {{appName}}</option>
```

Comment:
The repository names do not include space, so would be better to remove the spaces for the server name too.
```suggestion
          <option value="{{uuid}}">{{baseUrl}}/{{appName}}</option>
```